js/index.js

```diff
@@ -1 +1,10 @@
 // Your code goes here
+let funBus = document.querySelector('img');
+funBus.addEventListener('click', event => {
+    console.log('Fun bus was clicked!');
+});
+
+let mainNav = document.querySelectorAll('a');
+mainNav[0].addEventListener("mouseover", event => {
```

Comment:
If you would like for the underline text decoration to affect all of the a tags, you may need to use a forEach method.  Keep up the work and finish out the event listeners.